### DISCUSSION TITLE
Responsive containers that stop at the breakpoint

### DISCUSSION THREAD

**Discussion by simonschaufi** [twbs/bootstrap#41027] [Ideas] [answered]
Hi, I would like to propose a responsive container that stops at the defined breakpoint. This is kind of a mix between `.container` and `.container-fluid`. So instead of scaling all the way up to a certain breakpoint without any jumps, I would actually have jumps from breakpoint to breakpoint until a maximum breakpoint.

So to show some code:

`.responsive-container-lg` will behave like a normal `.container` until the lg-breakpoint is reached and then doesn't scale up further.

How could this be implemented? The current logic for generating the container classes with sass is quite complicated to understand for me.

See https://github.com/twbs/bootstrap/blob/main/scss/_containers.scss for the current code

**Comment by patkon** [comment, marked as answer]
I'm not quite sure I understand what you want to achieve. But couldn't your example be realized in this way?
`<div class="container" style="max-width:960px"> ... </div>`

If so, you could simply create your own max-width classes (mw-sm .. mw-lg .. mw-xxl ...) and then combine them.
`<div class="container mw-lg"> ... </div>`

**Reply by simonschaufi** [reply]
That is exactly what my intention was. Thank you for that easy solution. I don't see a need to integrate that into the framework.

**Reply by patkon** [reply]
You're welcome and Grüße nach Bretten
